### ISSUE TITLE
Blade formatter is not working

### ISSUE THREAD

**Issue by fartoot** [bajrangCoder/zed-laravel-blade#6]
The extension is not working, even after adding file_types to the settings:
`"file_types": {
  "Blade": ["*.blade.php"]
}`

Any help resolving this would be appreciated.
Thanks!

**Comment by bajrangCoder** [comment]
Acutally formatter doesn't works, seems something wrong in zed.
As I have defined the formatter here:
https://github.com/bajrangCoder/zed-laravel-blade/blob/7aba92bc95c50890ba2d4eb1e2f5aeb2e795fce2/languages/blade/config.toml#L26

**Comment by fartoot** [comment]
I’ve installed blade-formatter globally using:

`npm install -g blade-formatter
`

Then, I configured it as an external command like this:

```
"Blade": {
  "formatter": {
    "external": {
      "command": "blade-formatter",
      "arguments": ["{buffer_path}"]
    }
  }
}
```
It works perfectly!

**Comment by xcrap** [comment]
Yeah same with me, formatter not working, using the blade-formatter globally sometimes when I'm editing some files it just resets my changes and does not even let me edit, so I remove it...

**Comment by ZayRTun** [comment]
The following seems to be working perfectly for me.

```json
"languages": {
	"PHP": {
		"language_servers": [
			"intelephense",
			"!phpactor",
			"..."
		]
	},
	"Blade": {
		"formatter": {
			"external": {
				"command": "blade-formatter",
				"arguments": [
					"--stdin",
					"--wrap-line-length",
					"9999",
					"--wrap-attributes",
					"preserve-aligned"
				]
			}
		}
	}
},
"file_types": {
	"Blade": [
		"*.blade.php"
	]
}
```
